translator/translate/otel/processor/metricstransformprocessor/translator.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package metricstransformprocessor
+
+import (
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/processor"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+)
+
+type translator struct {
+	name    string
+	factory processor.Factory
+}
+
+var _ common.Translator[component.Config] = (*translator)(nil)
+
+func NewTranslatorWithName(name string) common.Translator[component.Config] {
+	return &translator{name, metricstransformprocessor.NewFactory()}
+}
+
+func (t *translator) ID() component.ID {
+	return component.NewIDWithName(t.factory.Type(), t.name)
+}
+
+func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
+	cfg := t.factory.CreateDefaultConfig().(*metricstransformprocessor.Config)
+	c := confmap.NewFromStringMap(map[string]interface{}{
+		"transforms": map[string]interface{}{
+			"include":                   "apiserver_request_total",
+			"match_type":                "regexp",
+			"experimental_match_labels": map[string]string{"code": "5.*"},
```

Comment:
does this need to be `^5.*`? I don't think it is common but I wonder if this would match (weird) http codes like `450`

Reply:
Update the regex to be more restrictive - ^5.*